ipycytoscape/cytoscape.py

```diff
@@ -191,6 +191,7 @@ def add_graph_from_networkx(self, g):
             receives a generic NetworkX graph. more info in
             https://networkx.github.io/documentation/
         """
+        self.directed = directed
```

Comment:
```suggestion
        self.directed = directed
        if self.directed:
            self.set_directed_style()
```

A potential problem here is if you call this function multiple times and then forget to call it with directed=True then self.directed will be reset to false.

Reply:
This doesn't actually work yet!

Reply:
Do you actually need something like this?

I ran the example you gave in the PR description and it worked fine. Once an user wants to convert a graph to a directed graph layout I think they could just call `set_directed_style()`.

What's your plan for this flag?